functions/HttpTriggerAPIUsersIdTask/__init__.py

```diff
@@ -40,26 +41,26 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
     logging.debug("Connected to DB successfuly!")
 
     #Redis Sever
-    try:
-        
-        rDB = redis.Redis(host='nsc-redis-dev-usw2-thursday.redis.cache.windows.net', port='6380', db=0, password= os.environ["ENV_REDIS_KEY"], ssl=True) 
-        rDB.ping()
-        logging.debug("Connected to Redis!")
-    except(redis.exceptions.ConnectionError, ConnectionRefusedError) as e:
-            logging.error("Redis connection error!" + e.args[0])
-            
 
+    if(redisFeature):
+        try:
+            rDB = redis.Redis(host='nsc-redis-dev-usw2-thursday.redis.cache.windows.net', port='6380', db=0, password='${{ secrets.ENV_REDIS_KEY }}', ssl=True) 

```

Comment:
the password needs to be os.environ["ENV_REDIS_KEY"], '${{ secrets.ENV_REDIS_KEY }}' is a reference to the github repo secrets.